tests/deriver/conftest.py

```
@@ -0,0 +1,401 @@
+"""Fixtures and test configuration for deriver tests."""
+
+import json
+import os
+from unittest.mock import AsyncMock, MagicMock, patch
+from uuid import uuid4
+
+import pytest
+import pytest_asyncio
+from nanoid import generate as generate_nanoid
```

Ruff (0.11.9)

`nanoid.generate` imported but unused

Remove unused import: `nanoid.generate`

(F401)